install.sh

```diff
@@ -1,21 +1,21 @@
 echo "================ Installing gradle ================="
-wget -nv https://services.gradle.org/distributions/gradle-4.2.1-all.zip
-unzip -qq gradle-4.2.1-all.zip -d /usr/local && rm -f gradle-4.2.1-all.zip
-ln -fs /usr/local/gradle-4.2.1/bin/gradle /usr/bin
-echo 'export PATH=$PATH:/usr/local/gradle-4.2.1/bin' >> /etc/drydock/.env
-
-echo "================ Installing apache-maven-3.5.2 ================="
-wget -nv http://redrockdigimark.com/apachemirror/maven/maven-3/3.5.2/binaries/apache-maven-3.5.2-bin.tar.gz
-tar xzf apache-maven-3.5.2-bin.tar.gz -C /usr/local && rm -f apache-maven-3.5.2-bin.tar.gz
-ln -fs /usr/local/apache-maven-3.5.2/bin/mvn /usr/bin
-echo 'export PATH=$PATH:/usr/local/apache-maven-3.5.2/bin' >> /etc/drydock/.env
-
-echo "================ Installing apache-ant-1.10.1 ================="
-wget -nv https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.1-bin.tar.gz
-tar xzf apache-ant-1.10.1-bin.tar.gz -C /usr/local && rm -f apache-ant-1.10.1-bin.tar.gz
-ln -fs /usr/local/apache-ant-1.10.1/bin/ant /usr/bin
-echo 'export ANT_HOME=/usr/local/apache-ant-1.10.1' >> /etc/drydock/.env
-echo 'export PATH=$PATH:/usr/local/apache-ant-1.10.1/bin' >> /etc/drydock/.env
+wget -nv https://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
why dont we first export the version in a variable up top and use that variable thereafter. no point in repeating this multiple times

Reply:
something like https://github.com/dry-dock/u16/blob/master/install.sh#L89

Reply:
ok